vignettes/inputs-layout.Rmd

```diff
@@ -62,7 +62,16 @@ By default inputs list is displayed with stacked labels. Stacked labels always a
 
 <br>
 
-The following inputs are supported: `f7Text`, `f7TextArea`, `f7Password`, `f7Select`, `f7SmartSelect`. This doesn't mean you can't use other inputs, but they may look a bit off. In this case you can take them out of `f7List` and style them manually with the available options in the input. For example:
+The following inputs are supported: 
+
+- `f7Text`
+- `f7TextArea`
+- `f7Password`
+- `f7Select`
```

Comment:
I think we can expand this list a little bit with a bit of work. It should be possible to get the range slider to work as well, and perhaps with a bit of effort also the pickers.

Reply:
`f7Stepper` could also work when the new layout parameter is `list`.

Reply:
`f7SmartSelect` renders also well:

```r
library(shiny)
library(shinyMobile)

shinyApp(
  ui = f7Page(
    options = list(dark = FALSE,
                   theme = "ios"),
    title = "Inputs Layout",
    f7SingleLayout(
      navbar = f7Navbar(
        title = "Inputs Layout",
        hairline = FALSE
      ),
      f7List(
        inset = TRUE,
        dividers = TRUE,
        strong = TRUE,
        f7Text(
          inputId = "text",
          label = "Text input",
          value = "Some text",
          placeholder = "Your text here"
        ),
        f7TextArea(
          inputId = "textarea",
          label = "Text area input",
          value = "Some text",
          placeholder = "Your text here"
        ),
        f7Select(
          inputId = "select",
          label = "Make a choice",
          choices = 1:3,
          selected = 1
        ),
        f7AutoComplete(
          inputId = "myautocomplete",
          placeholder = "Some text here!",
          openIn = "dropdown",
          label = "Type a fruit name",
          choices = c(
            "Apple", "Apricot", "Avocado", "Banana", "Melon",
            "Orange", "Peach", "Pear", "Pineapple"
          )
        ),
        f7Stepper(
          inputId = "stepper",
          "Stepper",
          0,
          10,
          5,
          layout = "list"
        ),
        f7SmartSelect(
          "smartselect",
          "SmartSelect",
          colnames(mtcars)
        )
      )
    )
  ),
  server = function(input, output) {
  }
)
```

Reply:
`f7Toggle` could be reworked to follow the list layout: https://framework7.io/docs/toggle#examples. This also a possible documented choice: https://framework7.io/docs/toggle#toggle-layout

Reply:
Thinking about it, but do we want to help users by checking whether or not the input is called in a list? I did this for the toggle now, but we can also do it for the stepper. Or do we want to have it like in the stepper with the `layout` param? Honestly I remembered far too late that stepper had this new param 🙃, but I do like the toggle solution because it automatically looks good in a list

Reply:
Maybe we could have the stepper with this new list layout and remove the layout parameter (which is not a breaking change because it was introduced in this PR), as long as this is consistent with other inputs.